src/python/datapreprocessor/datapreprocessor/entity.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+class EntityData():
```

Comment:
We're on python 2 so classes should inherit from object if they don't have another base class (this applies in a few places below too).